errors.go

```diff
@@ -58,12 +58,12 @@ func (a *App) PanicHandler(next Handler) Handler {
 			if r != nil { //catch
 				switch t := r.(type) {
 				case error:
```

Comment:
you might have to do `err = t` otherwise err is nill still

Reply:
good catch!